manifests/2.4.0/opensearch-2.4.0.yml

```diff
@@ -10,7 +10,7 @@ ci:
 components:
   - name: OpenSearch
     repository: https://github.com/opensearch-project/OpenSearch.git
-    ref: 2.x
+    ref: 2.4
```

Comment:
We would need single quote for this ref as lint setting.

Reply:
Fixed it

Reply:
Thanks!